pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/painless/UpsertPainless.java

```diff
@@ -46,7 +46,14 @@ public void export() {
                     "created/updated %s records in ES data for %s", getNumberOfUpserted(bulkByScrollResponse), generator.getPropertyName())
             );
         } catch (IOException e) {
-            throw new RuntimeException("Error occurred while exporting data to ES", e);
+            try {
```

Comment:
rather than duplicating this entire block, let's put it in a method of lambda.  so something like:
```
IntPredicate executeExport = (i) -> {
    try {
            BulkByScrollResponse bulkByScrollResponse = clientInstance.updateByQuery(updateByQueryRequest, RequestOptions.DEFAULT);
            logger.info(String.format(
                    "created/updated %s records in ES data for %s", getNumberOfUpserted(bulkByScrollResponse), generator.getPropertyName())
            );
           return true;
        } catch (IOException e) {
            throw new RuntimeException("Error occurred while exporting data to ES, on try number " + i, e);
       }
       return false;
}
for (int tryNum = 1; tryNum < 3; tryNum++) {
   if (executeExport(tryNum)) {
      break;
    }
}
```